README.md

```diff
@@ -0,0 +1,22 @@
+# EUCP Jupyterlab environment specification
+Dockerfile for the EUCP project - based on the Jupyter datascience notebook
```

Comment:
```suggestion
Dockerfile for the EUCP project - based on the Jupyter datascience notebook\
```
Assuming you want to start a new line after this. Options for that are </br>, double space (which is unclear in the raw text) or double enter which puts a blank line in between. Just one enter does not result in a new line in markdown.